servers/curated-corpus-api/src/admin/resolvers/mutations/SectionItem/index.ts

```diff
@@ -48,3 +49,34 @@ export async function createSectionItem(
 
   return sectionItem;
 }
+
+/**
+ * Removes an active SectionItem from a Section.
+ *
+ * @param parent
+ * @param args
+ * @param context
+ */
+export async function removeSectionItem(
+  parent,
+  args,
+  context: IAdminContext,
+): Promise<SectionItem> {
+  // First check if the SectionItem exists
+  const sectionItemToRemove =  await context.db.sectionItem.findUnique({
+    where: { externalId: args.externalId}
```

Comment:
what do you think about adding `active: true` to the `where` clause here? it's an edge case for sure, but might be a small change and could help avoid incorrect history on who deactivated a SectionItem.